tab_automl/automl/models.py

```diff
@@ -21,6 +22,7 @@
         "Decision Tree Classifier": DecisionTreeClassifier,
         "Light Gradient Boosting Classifier": LGBMClassifier,
         "Random Forest Classifier": RandomForestClassifier,
-        "XGBoost Classifier": XGBClassifier
+        "XGBoost Classifier": XGBClassifier,
+        "KNN classifier":KNeighborsClassifier
```

Comment:
Make the `c` capital for the `classifier` in the dictionary key.

Reply:
> Make the `c` capital for the `classifier` in the dictionary key.

yeah will do it